client-src/elements/chromedash-admin-blink-page.js

```diff
@@ -0,0 +1,212 @@
+import {LitElement, css, html} from 'lit';
+import {showToastMessage} from './utils.js';
+import {SHARED_STYLES} from '../sass/shared-css.js';
+import {VARS} from '../sass/_vars-css.js';
+import {LAYOUT_CSS} from '../sass/_layout-css.js';
+import './chromedash-admin-blink-component-listing';
+
+export class ChromedashAdminBlinkPage extends LitElement {
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      VARS,
+      LAYOUT_CSS,
+      css`      
+      body {
+          scroll-behavior: smooth;
+      }
+
+      #spinner {
+          display: none !important;
+      }
+
+      #subheader .subheader_toggles {
+          display: flex !important;
+          justify-content: flex-end;
+          flex: 1 0 auto;
+      }
+
+      #subheader ul {
+          list-style-type: none;
+          margin-left: var(--content-padding);
+      }
+
+      #subheader ul li {
+          text-align: center;
+          border-radius: var(--default-border-radius);
+          box-shadow: 1px 1px 4px var(--bar-shadow-color);
+          padding: .5em;
+          background: var(--chromium-color-dark);
+          color: #fff;
+          font-weight: 500;
+          text-transform: uppercase;
+      }
+
+      #subheader ul li a {
+          color: inherit;
+          text-decoration: none;
+      }
+
+      #subheader .view_owners_linke {
+          margin-left: var(--content-padding);
+      }
+
+      #components_list {
+          list-style: none;
+          margin-bottom: calc(var(--content-padding) * 4);
+      }
+
+      #components_list li {
+          padding: var(--content-padding) 0;
+      }`];
+  }
+  static get properties() {
+    return {
+      loading: {type: Boolean},
+      user: {type: Object},
+      _client: {attribute: false},
+      _editMode: {type: Boolean},
+      components: {type: Array},
+      usersMap: {type: Object},
+    };
+  }
+
+  /** @type {import('chromestatus-openapi').DefaultApiInterface} */
+  _client;
+
+  /** @type {Array<import('chromestatus-openapi').OwnersAndSubscribersOfComponent>} */
+  components;
+
+  /** @type {Map<int, import('chromestatus-openapi').ComponentsUser} */
```

Comment:
Does this need a closing angle bracket `>`?

Reply:
Good catch. Somehow auto-complete still worked without it but definitely want to add it for correctness.

![image](https://user-images.githubusercontent.com/7788930/227641013-7cce3c6b-ba99-41f6-a66f-b3630a5200a0.png)